src/instruments/src/RMP/Components/BaseRadioPanels.tsx

```diff
@@ -97,14 +97,16 @@ const PoweredRadioPanel = (props: Props) => {
         }
     });
 
-    // Not enough references to make MLS work
-    // Simulating the MLS system is connectedt to the ILS
-    useInteractionEvent(`A32NX_RMP_${props.side}_MLS_BUTTON_PRESSED`, () => {
-        if (navButtonPressed) {
-            setPanelMode(8);
-            setNavTransceiverType(TransceiverType.ILS);
-        }
-    });
+    /**
+     * MLS IMPLEMENTED IN THE XML BEHAVIOURS
+     * BUT DISABLED HERE SINCE THERE IS NOT ENOUGH REFERENCES
+     */
+    // useInteractionEvent(`A32NX_RMP_${props.side}_MLS_BUTTON_PRESSED`, () => {
```

Comment:
And MLS does not seem to actually exist anymore in the world 😛. At best we could allow "tuning" it but actually do nothing other than display the digits because there are no MLS transmitters to receive.